test/e2e/docker/Dockerfile

```diff
@@ -1,7 +1,7 @@
 # We need to build in a Linux environment to support C libraries, e.g. RocksDB.
 # We use Debian instead of Alpine, so that we can use binary database packages
 # instead of spending time compiling them.
-FROM golang:1.20
+FROM golang:1.20-bullseye
```

Comment:
Do we want to leave this change in the branch?

Reply:
I will rebase it away.

Reply:
Nevermind, check #984

Reply:
I'm leaving this in so as to not go through another merge of main into `feature/proto-upgrade` just for this.